client/src/css/meditation.css

```diff
@@ -0,0 +1,132 @@
+/* meditation.css */
+
+.container {
+    min-height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    font-family: 'Quicksand', sans-serif;
+    padding: 20px;
+    position: relative;
+    overflow: hidden;
+  }
+  
+  .cloud {
+    position: absolute;
+    top: 10%;
+    left: 20%;
+    width: 300px;
+    height: 150px;
+    background-color: white;
+    border-radius: 50%;
+    opacity: 0.8;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    z-index: -1;
+    animation: moveClouds 60s linear infinite;
+  }
+  
+  .meditation-content {
+    text-align: center;
+    background-color: rgba(255, 255, 255, 0.9);
+    border-radius: 30px;
+    padding: 60px;
+    box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+    width: 90%;
+    max-width: 600px;
+    animation: fadeIn 1s ease-out;
+  }
+  
+  .title {
+    font-size: 48px;
+    color: #4c9ed9;
+    margin-bottom: 30px;
+    font-weight: bold;
+    font-family: 'Nunito', sans-serif;
+  }
+  
+  .content {
+    font-size: 24px;
+    color: #555;
+    margin-bottom: 40px;
+    line-height: 1.5;
+    font-family: 'Quicksand', sans-serif;
+  }
+  
+  .button {
+    background-color: white;
+    color: #4c9ed9;
+    padding: 25px 50px;
+    font-size: 24px;
+    font-weight: bold;
+    border-radius: 30px;
+    cursor: pointer;
+    box-shadow: 0 4px 15px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+    width: 300px;
+    margin-top: 30px;
+    font-family: 'Quicksand', sans-serif;
+    border: none;
+    outline: none;
+  }
+  
+  .button:hover {
+    transform: scale(1.1);
+  }
+  
+  .arrow-container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    max-width: 600px;
+  }
+  
+  .arrow {
+    background-color: white;
+    color: #4c9ed9;
+    padding: 20px;
+    font-size: 28px;
+    font-weight: bold;
+    border-radius: 50%;
+    cursor: pointer;
+    box-shadow: 0 4px 15px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+    margin: 20px;
+    border: none;
+    outline: none;
+  }
+  
+  .arrow:hover {
+    transform: scale(1.1);
+  }
+  
+  .home-button {
+    position: absolute;
+    top: 20px;
+    right: 20px;
+    background-color: black;
+    color: white;
+    padding: 15px 25px;
+    font-size: 20px;
+    font-weight: bold;
+    border-radius: 5px;
+    cursor: pointer;
+    box-shadow: 0 4px 15px rgba(0, 0, 0, 0.1);
+    transition: all 0.3s ease;
+    border: none;
+    outline: none;
+  }
+  
+  .home-button:hover {
+    background-color: #4c9ed9;
+    transform: scale(1.1);
+  }
+  
+  .thank-you {
+    font-size: 28px;
+    color: #ffb347;
+    font-weight: bold;
+    font-family: 'Nunito', sans-serif;
+    padding-top: 30px;
+  }
+  
```

Comment:
hm its rly weird that it thinks you dont have a newline here. is there a space on line 132?

Reply:
okay it should be fixed now i believe